src/hooks/useSections.ts

```diff
@@ -51,15 +51,17 @@ export default function useSections(
   );
 
   // Fetch Recommendations Results
-  const recommendationsResults = useFetchRecommendationPod(cioClient, recommendationsSections);
-
+  const { recommendationsResults, podsData } = useFetchRecommendationPod(
+    cioClient,
+    recommendationsSections
+  );
   // Add to active sections the results data and refs when autocomplete results or recommendation results fetched
   useEffect(() => {
     const sectionsResults = { ...autocompleteResults, ...recommendationsResults };
     setActiveSectionsWithData(
-      getActiveSectionsWithData(activeSections, sectionsResults, sectionsRefs)
+      getActiveSectionsWithData(activeSections, sectionsResults, sectionsRefs, podsData)
```

Comment:
Here is the place where we can get the data from recommendationsResults where the display name on the items and add it the appropriate active section instead of returning podsData then passing it